helm/ph-ee-test/values.yaml

```diff
@@ -216,15 +193,34 @@ ph-ee-engine:
 
   channel:
     enabled: true
-    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-channel-test"
-    imageTag: test
+    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-channel"
+    imageTag: v1.5.1
     SPRING_PROFILES_ACTIVE: "bb"
     hostname: "channel2.sandbox.fynarfin.io"
     stub_hostname: "channel-gsma2.sandbox.fynarfin.io"
     DFSPIDS: "rhino,gorilla"
+    redis:
+      host: "127.0.0.1"
+      port: 6379
+      password: ""
+      idempotency:
+        enabled: true
+        keyFormat: "clientCorrelationId_tenant_api"
+        apiList: "/channel/transfer,/channel/collection,/channel/gsma/transaction,/channel/transactionRequest"
     zeebe:
       broker:
         contactpoint: "zeebetest-zeebe-gateway:26500"
+    operations:
+      url: "http://ops-bk.sandbox.fynarfin.io/api/v1"
```

Comment:
A lot of these contactpoints and hostnames are not matching with integration test hostnames / contact points.
Ex:- Integration test config https://ops-bk2.sandbox.fynarfin.io/ is not matching with contact point & hostname https://ops-bk.sandbox.fynarfin.io/. Keep https://ops-bk2.sandbox.fynarfin.io/ consistent.